### PR TITLE
Creating a new file

### LINKED ISSUE #3: add a file for azeez hometown

### PR DESCRIPTION
**Pull request by azeezmeister**
This PR resolves #3 by adding a file

### CODE REVIEW

File: ocala-fl.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### My hometown is famous for nothing
```
Comment (azeezmeister) [inline ocala-fl.md:1]: Adding a line comment

**Comment by sindhunair** [comment]
Can you add a new line to your file?

**Comment by azeezmeister** [comment]
This is looking :sparkle:  go ahead and :+1:

**Comment by sindhunair** [comment]
Looks good, go ahead and merge

**Pull request merged**